_includes/configuration/content-langs.md

```diff
@@ -0,0 +1,69 @@
+## `content_langs`
+
+{{ site.requires_5_9v }}
+
+This option allows you to override the languages displayed in the `language` dropdown toolbar button and the `language` menu item. This option accepts an array of language objects.
+
+Each language object is a JavaScript object defined using the following properties.
+
+| Name | Type | Requirement | Description |
+| ---- | ---- | ----------- | ----------- |
+| title | String | required | The name shown for this language in the UI. |
+| code | String | required | The language tag, as defined in [Tags for Identifying Languages (BCP47)](https://www.ietf.org/rfc/bcp/bcp47.txt), for this language. This tag is placed in the `lang` attribute for elements formatted using this language. |
+| customCode | String | optional | An alternative language tag, that does not need to be compatible with BCP47. This code is placed in the `data-mce-lang` attribute for elements formatted using this language, if it is defined. |
+
+**Type:** `Array<ContentLang>`
+
+**Default Value:**
+```js
+[
+  { title: 'English', code: 'en' },
+  { title: 'Spanish', code: 'es' },
+  { title: 'French', code: 'fr' },
+  { title: 'German', code: 'de' },
+  { title: 'Portuguese', code: 'pt' },
+  { title: 'Chinese', code: 'zh' }
+]
+```
+
+### Example: using `content_langs`
+
+```js
+tinymce.init({
+    selector: 'textarea', // change this according to your HTML
+    toolbar: 'language',
+    content_langs: [
```

Comment:
An example should always use something different to the default.

Reply:
There no longer is a default (unless you're using Spell Checker Pro, in which case the default is equivalent to this but it isn't actually this) so is that okay now?